### PR TITLE
[clkmgr] Exclude jitter enable from csr tests

### PR DESCRIPTION
**Pull request by tjaychen**
- If jitter enable is written with a random value, it may cause a prim_mubi4_sync failures.

- This is because the prim_mubi4_sync expect any non-false/true values are a one cycle transient.  A write to a different value clearly violates this and will need to be excluded.

Signed-off-by: Timothy Chen <timothytim@google.com>

### CODE REVIEW

**Comment by tjaychen** [comment]
related to the failure in #11196

**Review by a-will — APPROVED** [review]

**Review by msfschaffner — APPROVED** [review]
Looks like this needs a `make -C hw` pass, otherwise LGTM.

**Comment by tjaychen** [comment]
> Looks like this needs a `make -C hw` pass, otherwise LGTM.

i'll merge this once basic checks pass since it gates your other fix.

**Comment by msfschaffner** [comment]
> > Looks like this needs a `make -C hw` pass, otherwise LGTM.
> 
> i'll merge this once basic checks pass since it gates your other fix.

Sounds good!

**Pull request merged**
